ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -6,47 +6,53 @@
 import FluentUI
 
 class BottomSheetDemoController: DemoController {
-// MARK: Lifecycle methods
 
     override func viewDidLoad() {
         super.viewDidLoad()
```

Comment:
I still vote we should be using a loadView instead of viewDidLoad since we're constructing the hierarchy. viewDidLoad is usually meant for additional configuration after loading from a xib

Reply:
The reason was that the DemoController superclass adds a default stack view in viewDidLoad which we don't use, but it overlays everything and blocks interaction. I ended up just not using DemoController because we don't really use anything from it.